### PR TITLE
Less ignoring for mypy

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
setup.cfg currently makes mypy shut up about a lot of things.
This tries to reduce that, which also lead to some actual improvements. Note that before #2505 this would have been way harder

### Checklist for PRs

None of those really matter here …

- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [ ] Created new or adapted existing unit tests
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -70,6 +71,3 @@ warn_unused_ignores = False
 
 [mypy-urllib3.*]
 ignore_missing_imports = True
-
-[mypy-apscheduler.*]
-ignore_missing_imports = True
```
Comment (harshil21) [inline setup.cfg:75]: Why is this removed? I ran `mypy telegram/` on my machine and now it complains about missing type hints / stubs

**Review by harshil21 — COMMENTED** [review]
just a single comment

File: setup.cfg (hunk 1 above)
Comment (Bibo-Joshi) [reply setup.cfg]: mh, interesting that it doesn't complain during pre-commit … anyway, I reverted :)

**Review by harshil21 — APPROVED** [review]
LGTM! Deepsource has more false positives again..
